govcd/vm.go

```diff
@@ -832,6 +832,21 @@ func (vm *VM) GetProductSectionList() (*types.ProductSectionList, error) {
 	return getProductSectionList(vm.client, vm.VM.HREF)
 }
 
+// GetEnvironment returns the OVF Environment. It's only available for poweredOn VM
+func (vm *VM) GetEnvironment() (*types.OVF_Environment, error) {
```

Comment:
This function is basically doing a Refresh. So, why not calling VM.Refresh in the first place?
Moreover, I saw from the test that the environment is not available when the VM is not powered ON.
The way I understand it, the function should check if the VM is powered ON, return an error is it is not, and get the environment data when it is. No need to run a refresh, unless I am missing something.